plugins/modules/zos_copy.py

```diff
@@ -563,6 +563,61 @@
     returned: success
     type: str
     sample: SAMPLE.SEQ.DATA.SET
+dest_created:
+    description: Boolean value that indicate the creation of dest.
+    returned: success and if dest is create by the module.
+    type: bool
+    sample: true
+destination_attributes:
+    description: Values of a dest create by the module.
+    returned: success and values of the dest created by module.
```

Comment:
Wouldn't this information also be returned when "success and destination was created by the module"? I may be wrong on this one.